back/engines/free/email_campaigns/app/views/email_campaigns/project_phase_started_mailer/campaign_mail.mjml

```diff
@@ -30,3 +14,19 @@
 </mj-section>
 
 <%= render partial: 'application/cta_button', locals: { href: event.phase_url , message: format_message('cta_view_phase') } %>
+
+<mj-section padding="0 25px 25px"
```

Comment:
If this code is the same in the second place, could we move it to a partial?

Reply:
@alexander-cit I re-requested your review as I have now crated and used a partial, as you suggested.